### PR TITLE
StringConverter: Inline default argument of `list_delimiters` into function using a tuple

### PR DESCRIPTION
**Pull request by RohanVB**
Changed default arg into function using tuple

Fixes<https://github.com/coala-analyzer/coala/issues/2376>

### CODE REVIEW

**Comment by mr-karan** [comment]
not the PR title but commit message. HEAD should be short

https://coala.readthedocs.io/en/latest/Getting_Involved/Writing_Good_Commits.html

File: coalib/misc/StringConverter.py [hunk 1]
```diff
@@ -16,11 +16,9 @@ class StringConverter:
     def __init__(self,
                  value,
                  strip_whitespaces=True,
```
Comment (mr-karan) [inline coalib/misc/StringConverter.py]: space around `=`, do for the following occurrences as well.

Comment (RohanVB) [inline coalib/misc/StringConverter.py]: im on it. 1 min.

**Comment by Adrianzatreanu** [comment]
hey @RohanVB to fix all gitmate problems (coding style and so on) simply run ``coala`` and it will run over your code and fix all possible problems gitmate could find here :)

File: coalib/misc/StringConverter.py [hunk 2]
```diff
@@ -15,12 +15,10 @@ class StringConverter:
 
     def __init__(self,
                  value,
-                 strip_whitespaces=True,
-                 list_delimiters=None,
-                 dict_delimiter=":",
-                 remove_empty_iter_elements=True):
-        if list_delimiters is None:
```
Comment (Asnelchristian) [inline coalib/misc/StringConverter.py]: you can solve these by running `coala`

**Comment by Adrianzatreanu** [comment]
hey you have added the merge commit as well, you shouldnt do that. to fix it, do a ``git rebase -i origin`` and save and repush, that should be all :)

File: coalib/misc/StringConverter.py (hunk 1 above)
Comment (Adrianzatreanu) [inline coalib/misc/StringConverter.py]: 1) i dont think there should be spaces near operator, according to pep, or else gitmate would tell us?
2) it would be worth another commit for that change @mr-karan , probably

**Comment by mr-karan** [comment]
ack https://github.com/coala-analyzer/coala/pull/2399/commits/6cba64ee91ee6e329e235eb6ebc7bd98e317a794

**Comment by mr-karan** [comment]
@rultor merge
